logger.js

```diff
@@ -13,7 +13,7 @@ function pinoLogger (opts, stream) {
   opts.serializers.req = opts.serializers.req || asReqValue
   opts.serializers.res = opts.serializers.res || pino.stdSerializers.res
 
-  var logger = wrapChild(opts, stream)
+  var logger = wrapChild(opts, opts.stream || stream)
```

Comment:
I'm fine with that change, but keep attention, this is passing the stream to the pino instance. That might cause side effects once that api changes.

I'd directly set `opts.stream` instead of using the stream variable fallback so pino always receives the same configuration. No matter whether the stream instance is passed as second arg or on `opts`

```
opts.stream = opts.stream || stream
```